### ISSUE TITLE
Accessibility: Selection in MultiSelect is not accessible for screenreader

### ISSUE THREAD

**Issue by mehmetcetin01140** [primefaces/primeng#15930]
The selected options of a p-multiSelect are not readable by the screenreader.
If you tab on the multiselect field with a screenreader, the value of the field always seems to be empty no matter if an option is selected or not.
